### DISCUSSION TITLE
How can I port a Milvus DB from one VM to a different VM?

### DISCUSSION THREAD

**Discussion by abhirupghosh** [milvus-io/milvus#36390] [Q&A and General discussion]
I am presently running Milvus on an AWS VM - however, we wish to migrate to a different VM. Since we've run the indexing process on the VM already, how can I move it to the new VM without having to run the indexing process again?

**Comment by yhmo** [comment]
If your milvus is a standalone deployed by docker-compose, it is easy to migrate it.
The standalone will create a sub-directory named "volumes" under the same path of the docker-compose.yaml. All the data is stored in this "volumes". You can migrate it with the following steps:
1. `docker compose down` to stop/remove the containers(including milvus, etcd, minio), the "volumes" is still there
2. copy the "volumes" and the docker-compose.yaml to another machine
3. `docker compose up -d` to start new containers(including milvus, etcd, minio), the new containers will reuse the data in the "volumes", no need to re-index.

**Reply by abhirupghosh** [reply]
Okay - but one of my VMs has a GPU (this is the VM it was indexed on) so we were running Milvus with GPU, but we are moving to a VM without a GPU. Will that make a difference?

**Reply by yhmo** [reply]
GPU index must require a GPU device. If you move the GPU index to a machine without GPU, the milvus might not able to restart, or load fail.
Delete the GPU index by drop_index() before the migration.
